src/components/navigation-header/components/dropdown-menu/dropdown-menu.module.css

```diff
@@ -27,6 +27,9 @@
 	padding-left: var(--header-menu-item-padding-left-right);
 	padding-right: var(--header-menu-item-padding-left-right);
 	padding-top: 8px;
+	@nest html[data-theme='dark'] & {
+		color: blue;
+	}
```

Comment:
🚧 ⛏️ I think this might be some mid-dev cruft to clean up?

Reply:
Nice catch on the mobile nav! Let me give it another once over real quick to see if I missed anything else.

Reply:
Shoot! will fix that up. Thank you 🙇🏻